.travis.yml

```diff
@@ -17,6 +22,7 @@ notifications:
     on_failure: always
 script:
 - bundle exec rake default
```

Comment:
I think we need to change this to `bundle exec rake quick` like this: https://github.com/sensu-plugins/sensu-plugins-load-checks/blob/master/.travis.yml#L20. The idea here is to avoid duplicated integration runs.